src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -1768,15 +1638,9 @@ public XMPPError onTransportAccept(
         JingleSession jingleSession,
         List<ContentPacketExtension> contents)
     {
-        jingleSession.setAccepted(true);
-
         logger.info("Got transport-accept from: " + jingleSession.getAddress());
 
-        // We basically do the same processing as with transport-info by just
-        // forwarding transport/rtp information to the bridge
-        onTransportInfo(jingleSession, contents);
```

Comment:
Do we update the participant's local bridge anywhere now?

Reply:
`onTransportInfo` seems like a strict subset of `onSessionAccept` (if you unwrap `updateColibriChannels`).